libs/dh/market-participant/actors/feature-delegation/src/lib/create/dh-delegation-create-modal.component.ts

```diff
@@ -150,7 +149,7 @@ export class DhDelegationCreateModalComponent extends WattTypedModal<DhActorExte
 
     this.isSaving.set(true);
 
-    this._apollo
+    this.apollo
```

Comment:
Suggestion: Use mutation here

Reply:
This component will be refactored in a follow-up PR.